kubernetes/annotations_and_labels.md

```diff
@@ -47,13 +47,24 @@ This page defines common annotations and labels we set in Kubernetes objects.
   by the operator to recognize which object it should reconcile. When set on
   nodes it is used to set that information in the status with the
   statusresource.
+- `OPERATOR.giantswarm.io/branch` - (informational) branch from which this
+  instance of the `OPERATOR` was built from.
+- `OPERATOR.giantswarm.io/commit` - (informational) ID of the commit from which
+  this instance of the `OPERATOR` was built from.
```

Comment:
```suggestion
- `giantswarm.io/commit` - (informational) sha of the commit from which
  code was built from.
```

Reply:
How about `app.giantswarm.io/commit`?

Reply:
Then we know this is about the app and it isn't dynamic.

Reply:
Done, thanks!